ampeg/src/test/java/ManualDownloadsPageTests.java

```diff
@@ -0,0 +1,14 @@
+import org.testng.annotations.Test;
+import static org.testng.Assert.assertEquals;
+
+public class ManualDownloadsPageTests extends AmpegTests {
+	@Test
+	public void ensurev4BManualpdfLoads() {
+		var expectedurl = "https://ampeg.com/data/6/0a020a4177de6008ae7026058/application/pdf/Quick%20Start%20Guide%20-%20English%20.pdf";
```

Comment:
0a020a4177de6008ae7026058
This appears to be a dynamic value. Are you sure it won't change?

Reply:
hmmm, i see your point. Should I deprecate expectedurl and just put this.driver.getCurrentUrl() in the assertion?

Reply:
I would assert on the file name since it probably will not change.